### ISSUE TITLE
Schedule and multiple flottbot instances

### ISSUE THREAD

**Issue by wass3r** [target/flottbot-docs#23]
Call out in docs that the scheduler really only works with single bot instances

**Comment by mackenziegrimes** [comment]
While you're at updating docs, can you update the docs/example rules to have a six-argument cronjob schedule pattern? 

In my experience, a bot will run with a `second-minute-hour-etc` pattern, not `minute-hour-etc` as the docs mention. Maybe both are supported?

Anyway, the cron library that flottbot uses seems to have 6 args https://github.com/robfig/cron/blob/61d93e07d1be270591d43ede700aa3ae69a2326f/parser.go#L120

**Comment by wass3r** [comment]
@mackenziegrimes yes, will do. we have a related issue about that here: https://github.com/target/flottbot/issues/20

**Comment by wass3r** [comment]
it's been a minute, but the lib we're using got an update in the meantime.. i have something in queue to support both patterns, standard and quartz. i'll submit a PR for respective doc edits as well.
